### DISCUSSION TITLE
Can't seem to send options to nvim-ts-autotag anymore

### DISCUSSION THREAD

**Discussion by simeng** [LazyVim/LazyVim#3670] [Q&A] [answered]
I am asking here because I'm not sure if this is a bug or not. 
I don't really like autoclosing tags, but I like the rename-functionality of the plugin so I have lua/plugins/autotag.lua with
```
return {
  {
    "windwp/nvim-ts-autotag",
    event = "LazyFile",
    opts = {
      enable_close = false,
      enable_close_on_slash = false,
    },
  },
}
```
But the enable_close = false and enable_close_on_slash doesn't seem to do anything anymore, do I need to move this somewhere else for it to work or is it a bug?

**Comment by folke** [comment, marked as answer]
Check the docs.
should be opts = { opts = {...}}
